Dockerfile

```diff
@@ -9,15 +9,16 @@ RUN mkdir -p /var/lib/ironic-inspector && \
 
 RUN crudini --set /etc/ironic-inspector/inspector.conf DEFAULT auth_strategy noauth && \ 
     crudini --set /etc/ironic-inspector/inspector.conf ironic auth_strategy noauth && \
-    crudini --set /etc/ironic-inspector/inspector.conf DEFAULT debug true && \
+    crudini --set /etc/ironic-inspector/inspector.conf DEFAULT debug false && \
```

Comment:
just wondering if we should leave the debug level anyway

Reply:
Let's keep debug logging at this stage of development

Reply:
Changed debug back to true.